src/utils.ts

```diff
@@ -102,14 +96,16 @@ export const processWithOrdering = async <T>(
 
   const aggregateErrors = Object.values(unprocessedRecordsByListId)
     .map((record) => record.error)
+    .filter(Boolean)
```

Comment:
This filter seems odd to me. In what case would there be a falsy `error` in `unprocessedRecordsByListId`?

Reply:
The `unprocessedRecordsByListId` is initialized before actually calculating records, and we set the error to `null` as the init value.